Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -1,20 +1,18 @@
-// Name: Parent Processes running on Host
+// Id: 992b9912-a11e-487d-83b3-1d1ba6081746
+// DisplayName: Parent Processes running on Host
 // Description: Any 4688 event that contains Parent Process running on a given host during the time of or recently after compromise and in the range of +-10m
-// Entity: #Host
-// Input: Computername, Hostname, HostId
-// Output: Processes
-//
-// QueryPeriod: -1h and +30m default, change as needed, not all event 4688 have Parent Process available
-//
-// Data Source: #SecurityEvent
-//
+// InputEntityType: Host
+// InputFields: [HostName]
+// OutputEntityTypes: [Process]
+// QueryPeriodBefore: 10
+// QueryPeriodBeforeUnit: m
+// QueryPeriodAfter: 10
+// QueryPeriodAfterUnit: m
+// DataSources: [SecurityEvent]
```

Comment:
Again - why not align with the approach taken for queries under Detections folder?

Reply:
same, changed